### PR TITLE
Add missing edge case tests from old Portal

### PR DESCRIPTION
**Pull request by Piotrk39**
[AB#36773](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/36773) 

## Describe your changes

Old portal had some tests that were missed when transferring to Portalicious and we should add them here as well:

1. WrongLanguagesInTemplatedMessagesBug
Try to send a templated message to a registration that has a language other than english as their preferred language, and then validate in the activity log that they receive the message in their preferred language (and not English) 

2. SuccessfullyAddNote
Self-explanatory - successfully add a note in the registration profile page, and then view the note in the activity log

NOTE: Number 1 is already covered in "SendTemplatedMessage" test and that is why it is skipped in this PR

## Checklist before requesting a code review

- [x] I have performed a self-review of my code
- [x] I have added tests for my changes, or: Adding tests is unnecessary/irrelevant
- [ ] I have asked the design team to review these changes, or: The changes do not touch the UI/UX
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

## Portal preview-deployment

This PR does not have any preview deployments yet.

### CODE REVIEW

File: e2e/portal/pages/RegistrationBasePage.ts [hunk 1]
```diff
@@ -64,6 +64,13 @@ abstract class RegistrationBasePage extends BasePage {
     await this.clickActionDropdown();
     await this.page.getByRole('menuitem', { name: action }).click();
   }
+
+  async fillNote(note: string) {
+    const noteInput = this.page.locator('textarea');
```
[the remark this answers is not in the document]
Comment (Piotrk39) [reply e2e/portal/pages/RegistrationBasePage.ts:69]: Within this context locator appears only once so there is no need for extra data-test-id in this case.

File: e2e/portal/components/TableComponent.ts [hunk 2]
```diff
@@ -445,6 +445,14 @@ class TableComponent {
 
     await expect(messageNotification).not.toBeVisible();
   }
+
+  async validateActivityPresentByType(notificationType: string) {
+    const messageNotification = this.page
+      .locator('tr')
```
[the remark this answers is not in the document]
Comment (Piotrk39) [reply e2e/portal/components/TableComponent.ts:451]: Unfortunately this locator does not work like that in the primeNG table.

File: e2e/portal/tests/UpdateRegistrations/BulkUpdateVisibleInActivityLog.spec.ts [hunk 3]
```diff
@@ -70,10 +70,10 @@ test('[36352] Data should be updated according to selected columns and registrat
     await registrationsPage.goToRegistrationByName({
       registrationName: newName,
     });
-    // It is a soft assertion but the test will fail if the activity is not found
     await tableComponent.filterColumnByDropDownSelection({
       columnName: 'Activity',
       selection: 'Data change',
     });
+    await tableComponent.validateActivityPresentByType('Data change');
```
[the remark this answers is not in the document]
Comment (Piotrk39) [reply e2e/portal/tests/UpdateRegistrations/BulkUpdateVisibleInActivityLog.spec.ts:77]: Done!
